src/main/java/org/traccar/WebDataHandler.java

```diff
@@ -54,15 +57,38 @@
     private final String header;
     private final boolean json;
 
+    private final Boolean retryEnabled;
```

Comment:
Use primitives for all these configuration parameters.

Reply:
Ok.

Reply:
Sorry, should we use primitives also on `src/main/java/org/traccar/config/Keys.java`?
I just remembered that not using primitives there, was the reason why I didn't use primitives here.

Reply:
No, there we use class types.

Reply:
Hm... Ok then. Thanks.